packages/xo-web/src/xo-app/pool/tab-advanced.js

```diff
@@ -440,7 +441,17 @@ export default class TabAdvanced extends Component {
                         value={migrationNetwork}
                         xoType='network'
                       >
-                        {migrationNetwork !== undefined ? <Network id={migrationNetwork.id} /> : _('noValue')}
+                        {pool.otherConfig['xo:migrationNetwork'] === undefined ? (
+                          _('noValue')
+                        ) : migrationNetwork !== undefined ? (
+                          <Network id={migrationNetwork.id} />
+                        ) : (
+                          <span className='text-danger'>
+                            {_('networkIdMissing', {
+                              networkID: <Copiable>{pool.otherConfig['xo:migrationNetwork']}</Copiable>,
```

Comment:
What do you think to wrap the UUID with `<strong>`element?
```suggestion
                              networkID: (
                                <Copiable>
                                  <strong>{pool.otherConfig['xo:backupNetwork']}</strong>
                                </Copiable>
                              ),
```